src/app/child-dev-project/previous-schools/previous-schools.component.ts

```diff
@@ -51,7 +76,7 @@ export class PreviousSchoolsComponent
     }
 
     this.records = await this.childrenService.getSchoolRelationsFor(id);
-    this.current = this.records.find((record) => record.isActive);
+    this.hasCurrentlyActiveEntry = _.some(this.records, ['isActive', true]);
```

Comment:
```suggestion
    this.hasCurrentlyActiveEntry = this.records.some((rec) => rec.isActive);
```
I guess this makes it easier to understand for people unfamiliar with the `lodash` library